src/markdown-pages/build-apps/add-visualizations-to-dashboard.mdx

```diff
@@ -66,7 +66,7 @@ Read our [Nerdpacks privileges](/build-apps/permission-manage-apps/) documentati
 
 </Callout>
 
-Next, find your published visualization in the New Relic One app catalog.
+Next, find your published visualization in Instant Observability.
```

Comment:
```suggestion
Next, find your published visualization in the New Relic One app overlay.
```

Think this one needs to point to the app overlay. Not sure what language we're aligning on for that platform UI link, but `Instant Observability` doesn't match the next step

![Screen Shot 2021-09-29 at 11 38 30 AM](https://user-images.githubusercontent.com/20293876/135329480-741b1251-2558-4709-9012-18476d8daccd.jpg)

Reply:
https://developerwebsitedevelop-mlosieriopackdetailsrefresh.gtsb.io/build-apps/add-visualizations-to-dashboard/